mycore-ocfl/src/main/java/org/mycore/ocfl/MCROCFLXMLMetadataManager.java

```diff
@@ -175,7 +178,8 @@ private String getObjName(MCRObjectID mcrid) {
     }
 
     private String getObjName(String mcrid) {
-        return MCR_OBJECT_ID_PREFIX + mcrid;
+        String objectType = MCRObjectID.getIDParts(mcrid.trim())[1].toLowerCase(Locale.ROOT).intern();
```

Comment:
Could use MCRObjectID.getInstance(mcrid).getTypeId() here. It uses caching and the ID is probably already cached, so no string operation needed.